workflow/rules/common.smk

```diff
@@ -363,7 +364,7 @@ for p in patient_list:
     consensus.append(os.path.join(sdir, "references/ref_ambig.fasta"))
     consensus.append(os.path.join(sdir, "references/ref_majority.fasta"))
 
-    consensus.append(os.path.join(sdir, "references/consensus.bcftools.fasta"))
+    # consensus.append(os.path.join(sdir, "references/consensus.bcftools.fasta"))
```

Comment:
This should not be commented out, right?

Reply:
Yes, you are right.